### DISCUSSION TITLE
info: downloading installer Failure writing output to destination dgedb-init: command failed

### DISCUSSION THREAD

**Discussion by Tk8and15** [geldata/gel#4840] [Q&A] [answered]
info: downloading installer
Failure writing output to destination
dgedb-init: command failed
![image](https://user-images.githubusercontent.com/118907993/208813131-16691f59-ad0f-4d07-9374-f933f88e458c.png)
![image](https://user-images.githubusercontent.com/118907993/208813160-3a5152af-98f1-44fa-bd4f-9e843231d2fd.png)
How to use edgedb cli after downloading it?

**Comment by tailhook** [comment, marked as answer]
1. Fixing script error: you should make sure that `mktemp -d` returns a valid writable directory with enough space on the filesystem to download the file. You can customize that via setting `$TMPDIR`:
    ```shell
    mkdir -p ~/.tmp
    export TMPDIR=~/.tmp
    curl --proto '=https' --tlsv1.2 -sSf https://sh.edgedb.com | sh
    ```
2. If you manually downloaded the binary you can put it into your home dir, making it executable, and ensure that it's in PATH:
    ```shell
    mkdir -p ~/.local/bin
    mv edgedb-cli-2.2-whatever-version-youve-donwloaded ~/.local/bin/edgedb
    chmod +x ~/.local/bin/edgedb
    ```
    With typical modern configs you now can exit the shell and start a new one, and `edgedb` command will be working. If it's not you can do:
    ```
    echo 'export PATH="$HOME/.local/bin:$PATH"' >> ~/.profile
    ```
    And restart the shell again, `edgedb` command should work in this case.
